csm/conf/setup.py

```diff
@@ -78,6 +78,31 @@ def _run_cmd(cmd):
         except Exception as e:
             raise CsmSetupError("Csm setup is failed Error: %s %s" %(e,_err))
 
+    @staticmethod
+    def _fetch_csm_user_password(decrypt=False):
+        """
+        This Method Fetches the Password for CSM User from Provisioner.
+        :param decrypt:
+        :return:
+        """
+        Log.info("Fetch CSM User Password from provisioner.")
+        csm_user_pass = None
+        csm_credentials = SaltWrappers.get_salt_call(const.PILLAR_GET, "csm")
+        if csm_credentials and type(csm_credentials) is dict:
+            csm_user_pass = csm_credentials.get(const.SECRET)
+        else:
+            Log.error("No Credentials Fetched from Provisioner.")
```

Comment:
If no credentials are fetched we should return from here. In this case we will save one if statement check.

Reply:
sure